wifi: Fix formatting issues

krish2718 opened the pull request:
Automatic format with `clang-format -i` as per rules in `.clang-format`.

If some formatting isn't allowed, then we need to fix the `.clang-format` and re-run.

krish2718 commented on subsys/net/l2/wifi/CMakeLists.txt:
```diff
@@ -1,49 +1,59 @@
-    ${ZEPHYR_BASE}/samples/net/wifi/test_certs/ca.pem
-    ${gen_dir}/ca.pem.inc
-    )
+#SPDX - License - Identifier : Apache - 2.0
```
this looks fishy.

jukkar commented on subsys/net/l2/wifi/wifi_shell.c:
```diff
@@ -31,35 +31,30 @@ LOG_MODULE_REGISTER(net_wifi_shell, LOG_LEVEL_INF);
-				NET_EVENT_WIFI_AP_DISABLE_RESULT  |\
-				NET_EVENT_WIFI_AP_STA_CONNECTED   |\
-				NET_EVENT_WIFI_AP_STA_DISCONNECTED)
+#define WIFI_SHELL_MGMT_EVENTS_COMMON                                                              \
+	(NET_EVENT_WIFI_SCAN_DONE | NET_EVENT_WIFI_CONNECT_RESULT |                                \
+	 NET_EVENT_WIFI_DISCONNECT_RESULT | NET_EVENT_WIFI_TWT | NET_EVENT_WIFI_RAW_SCAN_RESULT |  \
+	 NET_EVENT_WIFI_AP_ENABLE_RESULT | NET_EVENT_WIFI_AP_DISABLE_RESULT |                      \
+	 NET_EVENT_WIFI_AP_STA_CONNECTED | NET_EVENT_WIFI_AP_STA_DISCONNECTED)
```
I am not sure we should blindly do what the clang-format suggests. For example here the original code is much more readable.

jukkar commented on subsys/net/l2/wifi/wifi_shell.c:
```diff
@@ -145,31 +138,27 @@ static bool parse_number(const struct shell *sh, long *param, char *str,
 	if (context.scan_result == 1U) {
-		PR("\n%-4s | %-32s %-5s | %-13s | %-4s | %-15s | %-17s | %-8s\n",
-		   "Num", "SSID", "(len)", "Chan (Band)", "RSSI", "Security", "BSSID", "MFP");
+		PR("\n%-4s | %-32s %-5s | %-13s | %-4s | %-15s | %-17s | %-8s\n", "Num", "SSID",
+		   "(len)", "Chan (Band)", "RSSI", "Security", "BSSID", "MFP");
```
Here too, it makes lot of sense to have parameters in a separate line.

jukkar commented on subsys/net/l2/wifi/wifi_shell.c:
```diff
@@ -296,63 +279,44 @@ static void handle_wifi_disconnect_result(struct net_mgmt_event_callback *cb)
-	      twt_wake_interval);
-	PR("TWT interval: %lld us\n",
-	      twt_interval);
+	PR("TWT Dialog token: %d\n", dialog_token);
+	PR("TWT flow ID: %d\n", flow_id);
+	PR("TWT negotiation type: %s\n", wifi_twt_negotiation_type_txt(negotiation_type));
+	PR("TWT responder: %s\n", responder ? "true" : "false");
+	PR("TWT implicit: %s\n", implicit ? "true" : "false");
+	PR("TWT announce: %s\n", announce ? "true" : "false");
+	PR("TWT trigger: %s\n", trigger ? "true" : "false");
+	PR("TWT wake interval: %d us\n", twt_wake_interval);
+	PR("TWT interval: %lld us\n", twt_interval);
```
But for example here, the suggested output is easier to read.

jukkar commented on subsys/net/l2/wifi/wifi_shell.c:
```diff
@@ -491,27 +450,18 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
-		{"key-passwd", required_argument, 0, 'K'},
-		{"help", no_argument, 0, 'h'},
-		{0, 0, 0, 0}};
+		{"ssid", required_argument, 0, 's'},     {"passphrase", required_argument, 0, 'p'},
+		{"key-mgmt", required_argument, 0, 'k'}, {"ieee-80211w", required_argument, 0, 'w'},
+		{"bssid", required_argument, 0, 'm'},    {"band", required_argument, 0, 'b'},
+		{"channel", required_argument, 0, 'c'},  {"timeout", required_argument, 0, 't'},
+		{"anon-id", required_argument, 0, 'a'},  {"key-passwd", required_argument, 0, 'K'},
+		{"help", no_argument, 0, 'h'},           {0, 0, 0, 0}};
```
Not like this, the original is much more readable.

jukkar reviewed (changes requested):
I do not want we blindly apply the formatting changes, the output looks really bad in places.

krish2718 commented:
> I do not want we blindly apply the formatting changes, the output looks really bad in places.

I prefer modifying the clang-format file and then rely on auto update, this way format is consistent. Manual fixes should be avoided.

krish2718 commented:
> > I do not want we blindly apply the formatting changes, the output looks really bad in places.
> 
> I prefer modifying the clang-format file and then rely on auto update, this way format is consistent. Manual fixes should be avoided. Formatting is highly subjective, so, we should 100% automate it and then we can forget about it :)

@jukkar if we can agree on the approach I can redo this and submit. I still feel fully automated approach is better at the cost of few quirks.

Commit:
wifi: Automatic format using clang-format

Use clang-format to auto-format all Wi-Fi related files.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

Commit:
net: l2: wifi: Fix indent manually

This fixes a compliance of issue of crossing 115 lines due to using too
many tabs.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

Commit:
net: l2: wifi: Fix typo

Fix the compliance error.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>
